### ISSUE TITLE
new ARMS observatories

### ISSUE THREAD

**Issue by kmexter** [emo-bon/governance-data#23]
Depending on what exactly has been decided, we will need to add new ARMS observatories to the governance files
Likely to be added are those in rows 2-11 in the attached spreadsheet 
[Copy of EMOBON_ARMS-MBON_project_accessions_MO (1).xlsx](https://github.com/user-attachments/files/16248016/Copy.of.EMOBON_ARMS-MBON_project_accessions_MO.1.xlsx)

Matthias wants the two Ravs to be part of EMO BON, TZS, and AdriaticEastCoast  also - if so, then we need to chose an observatory ID, and create a project accession in ENA (@cpavloud would do this)

The organisations, observatories, and logsheets.csv files will need updating
The logsheets will need to be filled and QCd before harvesting

**Comment by cpavloud** [comment]
Should we wait for Nicolas to be back from vacation to see if he agrees with Matthias suggestion?

**Comment by kmexter** [comment]
yes - I will not be doing anything at all until I am back (>19 Aug) so there is time for conversations to be had

**Comment by kmexter** [comment]
This issue is waiting for completion of https://github.com/emo-bon/observatory-profile/issues/20. At that point I should add those observatories to the harvesting, work with Bram on getting the QC to include the ARMS (note: called "hard" in the logsheet schema) and with Laurian to get this included in the data model and the templates

**Comment by isanti** [comment]
Logsheet for TZS is already created. They are officially in EMO BON since Dec 2024.
For AdriaticEastCoast there is also a logsheet. We had this discussion or Bolognia in the past. Because they are EMBRC and they are collaborating with CNR-ISMAR which is in EMO BON, they are in.

Close this one?
The https://github.com/emo-bon/observatory-profile/issues/20 I will look later.

**Comment by kmexter** [comment]
For ARMS we need

1. to have the files in https://github.com/emo-bon/governance-data to contain all the EMO BON arms observatories, and (for now at least) not those that are not in emo bon, and to have the googlesheets for all observatories in the googledrive  (those ARMS observatories NOT in emo bon CAN also be in the googledrive, but they CANNOT be in the governance-data files)
2. to have an agreed way to refer to arms "observatories" that does not clash with the emo bon definition of "observatories" (that is part of https://github.com/emo-bon/observatory-profile/issues/20) 
3. to agree on the format for the ARMS logsheets (that is partly in https://github.com/emo-bon/observatory-profile/issues/20 also, see issues 43, 45 44 in that same repo also)

if 1 is done, then this issue can be closed, yes

**Comment by cpavloud** [comment]
The thing is what are we going to do with the ARMS station that are not EMO BON but have been sequenced in Genoscope (so EMBRC has covered their sequencing costs):
Galway (samples sequenced in EMO BON batch 3)
Bodo (samples sequenced in EMO BON batch 3)
Svalbard (samples sequenced in EMO BON batch 3)
Laeso (samples sequenced in EMO BON batch 3)
Nuuk (samples sequenced in EMO BON batch 4)
Rothera (samples sequenced in EMO BON batch 3)
GulfOfPiran (samples sequenced in EMO BON batch 3 & batch 5)
Gdynia (samples sequenced in EMO BON batch 3)

I have tried to summarize them [here](https://docs.google.com/spreadsheets/d/1uYJNqIGZN3NuIjBj8QkY-cM64HC37oCq04Rhr3AfwVM/edit?usp=sharing).

**Comment by cymon** [comment]
Should we avoid analysing them with metaGOflow (probably no where near the top of the list yet, but just checking...)?

**Comment by cpavloud** [comment]
> Should we avoid analysing them with metaGOflow (probably no where near the top of the list yet, but just checking...)?

No need to worry about it. ARMS are only metabarcoding samples, so metaGOflow doesn't apply.

**Comment by cymon** [comment]
Ah yes, of course...

**Comment by isanti** [comment]
Updated Christina's file [here](https://docs.google.com/spreadsheets/d/1uYJNqIGZN3NuIjBj8QkY-cM64HC37oCq04Rhr3AfwVM/edit?usp=sharing)

Added the ARMS EMO BON here https://github.com/emo-bon/governance-data in all three csv: observatories, organisations, logsheets

All logsheets are in the google drive (EMO BON + non-EMO BON)

This can close, I guess. And we deal with 2-3 in the relevant issues.
